src/views/UserEdit/UserEdit.js

```diff
@@ -265,7 +283,6 @@ class UserEdit extends React.Component {
     const prevActive = prevUser.active;
 
```

Comment:
to be consistent with rest of the code, place all const declarations in immediate lines.